### PR TITLE
gh-122744: test_venv: ignore pip's complaint about missing ssl

### PR DESCRIPTION
**Pull request by encukou**
This is a new warning emitted by pip for its [“Using system certificate stores” feature](https://pip.pypa.io/en/stable/topics/https-certificates/#using-system-certificate-stores).

* Issue: gh-122744

### CODE REVIEW

**Review by vsajip — APPROVED** [review]

**Comment by sethmlarson** [comment]
I can't speak for pip maintainers, but I do wonder if the phrasing of "truststore" is a hold-over from when it was an optional feature enabled by flag (ie `--use-feature=truststore`) instead of the actual feature name? Maybe the skipped warning can look for `ssl support is missing` instead of the full sentence?

**Comment by encukou** [comment]
Well, we can easily adjust whenever `pip` is updated again -- it'll be in a new CPython PR, anyway.
I'll merge now to unblock the buildbots, useing the full message to avoid guessing about the future.

**Pull request merged**
